tslint.ts

```diff
@@ -1,10 +1,74 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as coreModules from 'resolve/lib/core';
 import * as tslint from 'tslint';
 import * as tslintAirbnb from 'tslint-config-airbnb';
 const tslintReact = require('tslint-react');
 
 const rulesAirbnb = tslintAirbnb.rules;
 const rulesReact = tslintReact.rules;
 
+// prepare regex for node core modules for 'ordered-imports' rule
+const modules = Object.entries(coreModules as Record<string, boolean>).reduce(
+  (acc: string[], [module, enabled]) => {
+    enabled && !module.startsWith('_') && acc.push(module);
+    return acc;
+  },
+  []
```

Comment:
That's a misuse of reduce.
```
.filter(([module, enabled]) => enabled && !module.startsWith('_'))
.map(([module]) => module);
```

Reply:
Why are we filtering out modules starting with `_` btw?

Reply:
@prk3 why is it a misuse of reduce? reduce > filter/map always!

Reply:
IMO this is not a misuse - it prevents going through the list of modules twice, which for something that may run continuously in the background is not a bad thing

Reply:
Reduce (fold) is for reducing a list into a single value. Yes, that single value can be an array, but at this point we're basically using is as a `for..of` loop. The cost of iterating through a list of 100 elements twice is negligible (probably less than the cost of expanding array with .push - that's why Array has a constructor taking capacity btw).

Reply:
> 
> 
> Why are we filtering out modules starting with `_` btw?

there are modules like `_stream_duplex|_stream_transform|_stream_wrap|_stream_passthrough|_stream_readable|_stream_writable` which are not meant to be imported directly

Reply:
@prk3 @flipace would a `for..of` loop make more sense here?

Reply:
I tried `for..of` but I have no idea why typescript is yelling at me :/
![image](https://user-images.githubusercontent.com/14310995/64697205-91f32e00-d4a0-11e9-8edc-91278dbe5742.png)